INSTALL

```diff
@@ -85,3 +88,59 @@ copy and paste these lines into the shell:
 You should see lmi's GUI. Try "File | New | Illustration", then press "OK";
 you should see values. Then try "File | Print preview"; you should see an
 illustration in your 'pdf' viewer.
+
+
+Install using Windows Subsystem for Linux (Windows 10)
+---
+
+(1) Install WSL
+
+Open a "Windows PowerShell" window as Administrator:
+
+  Start | Programs | Windows PowerShell | Windows PowerShell
+
+Copy and paste the following lines into the "Windows PowerShell" window:
+
+  New-Item -ItemType Directory -Force -Path C:\cache_for_lmi
+  Set-Location -Path C:\cache_for_lmi
+  Invoke-WebRequest -Uri https://git.savannah.gnu.org/cgit/lmi.git/plain/install_wsl.ps1 -OutFile install_wsl.ps1 -UseBasicParsing
+  Set-ExecutionPolicy Unrestricted
+  .\install_wsl.ps1
+
+Agree to restart your computer, open a "Windows PowerShell" window as
+Administrator and execute the installation script again:
+
+  .\install_wsl.ps1
+
+Select preferred OS or simple press Enter to install Ubuntu 18.04.
```

Comment:
```suggestion
Select preferred OS or simply press Enter to install Ubuntu 18.04.
```

Also, selecting non-Debian-based OS won't work, as we use apt, so I think it should be mentioned here.